recipe/meta.yaml

```diff
@@ -18,19 +18,17 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - versioneer 0.28
   run:
     - python
-    - bokeh >=2.4.2,<3.0
+    - bokeh >=2.4.2
     - cytoolz >=0.12.0
     - dask-core {{ version }}
     - distributed {{ version }}
     - jinja2 >=2.10.3
     - lz4 >=4.3.2
     - numpy >=1.21
     - pandas >=1.3
-    - pyarrow >=7.0                  # [not s390x]
```

Comment:
Why remove pyarrow?

Reply:
I must have gotten trigger happy with the delete button. I've added it back now.